src/systemcmds/topic_listener/generate_listener.py

```diff
@@ -24,6 +24,15 @@
 for index,m in enumerate(raw_messages):
 	topic_list = []
 	f = open(m,'r')
+	has_timestamp = False
```

Comment:
A timestamp should be required in all messages. What is the rationale here?

Reply:
The idea was to leave deprecated uorb message defintions, as an empty file and remove it from the CMakeLists. In order for the listener_generator not to add empty, deprecated message definition, this check was added. This will be obsolete after adding the deprecated message list.

Reply:
What is the goal of leaving it as an empty file? I feel like it doesn't add any value and we can just remove it.

Reply:
This change can be removed now.